app/views/controllers/observations/show/_section_update.erb

```diff
@@ -1,26 +1,44 @@
+<%# locals: (identifier: "", obs: nil, user: nil, consensus: nil, sites: nil) %>
+
+<%#
+Generalized template for Turbo live partial updates of the show obs page.
+This template is rendered by several controllers upon success of any action
+that creates or updates a resource associated with the Observation - for
+example a Naming, CollectionNumber, FungariumRecord, ExternalLink, or Sequence.
+
+The reason this is a generalized template is because these updates should all
+do at least two things, via two separate `turbo_stream` actions below:
+  - Replace the relevant section of show obs
+  - Update the page flash to provide user feedback
+Each of the above controller responses could *explicitly* call both of these,
+but this partial is intended to be more DRY, at the cost of being indirect.
+
+Callers should pass the local_assign `identifier`, plus any locals needed by
+the section partial. The `identifier` is a string that should match both
+  - the partial filename: views/controllers/observations/show/_#{identifier}.erb
+  - the last segment of the section's HTML ID: "observation_#{identifier}"
+
+NOTE: The section partial should also have a `data-controller="status-update"`
+attribute on the outermost div. This calls the Stimulus controller to hide any
+modal forms when the div gets updated by Turbo.
```

Comment:
@JoeCohen @mo-nathan - this is an example of code/comments i was trying to improve and make more understandable.